librecad/src/main/qc_applicationwindow.cpp

```diff
@@ -129,6 +129,9 @@ QC_ApplicationWindow* QC_ApplicationWindow::appWindow = nullptr;
  *	*/
 #define WTB_MAX_SIZE        79
 
+#define DEFAULT_STATUS_BAR_MESSAGE_TIMEOUT 2000
```

Comment:
Using preprocessor macros for constants is deprecated.
In this case I think a `private` `enum` inside the class wold be best choice.
`enum`s should use camel case and start upper case.

Reply:
I did not know that. Is there a contributing developer's guidelines page that I could read through?

I wrote my code by referencing the style employed by other authors in the respective files.